bin/ghe-restore

```diff
@@ -350,11 +350,6 @@ if is_external_database_target_or_snapshot && $SKIP_MYSQL; then
 else
   echo "Restoring MySQL database from ${backup_snapshot_strategy} backup snapshot on an appliance configured for ${appliance_strategy} backups ..."
   ghe-restore-mysql "$GHE_HOSTNAME" 1>&3
-  # Clear GitHub Connect settings stored in the restored database
-  if ! $RESTORE_SETTINGS; then
-    echo "if [ -f /usr/local/share/enterprise/ghe-reset-gh-connect ]; then /usr/local/share/enterprise/ghe-reset-gh-connect -y; fi" |
```

Comment:
> if is_external_database_target_or_snapshot && $SKIP_MYSQL; then
  echo "Skipping MySQL restore."
 else
   echo "Restoring MySQL database from ${backup_snapshot_strategy} backup snapshot on an appliance configured for ${appliance_strategy} backups ..."<